cpp/ps0002/ps0002solution.cpp

```diff
@@ -4,138 +4,35 @@
+    unsigned short buildingResidents = 40; // 0.75/0.75
 
     //عدد البدايل لدوا في الصيدلية
-    short medicinAlternatives = 4;
+    unsigned short medicinAlternatives = 4; // 0.75/0.75
 
     //أفراد تيم في شركة برمجة
-    short teamMembers = 5;
+    unsigned short teamMembers = 5; // 0.75/0.75
 
     //سعر واحدة من مخبوزات الفرن
-    float bakedPrice = 10.5;
+    float bakedPrice = 10.5; // 0.75/0.75
 
     //الحرف الأول من اسم شخص
-    char personFirstLetter = 'h';
+    char personFirstLetter = 'h'; // 0.75/0.75
 
     //عدد المواليد في دولة تعدادها كبير زي الصين
-    long populationNumber = 2912534765;
+    unsigned int populationNumber = 2912534765; // 0.75/0.75
 
     //ناتج اكثر دقة لقسمة رقم على 7
-    double result = 83.8571429;
+    double result = 83.8571429; // 0.75/0.75
 
     //فاتورة الكهربا لخط انتاج في مصر
-    double electricityBill = 332188737363636.64564546;
+    double electricityBill = 332188737363636.64; // 0.75/0.75
 
     //عمر أشخاص كبار و صغار ف السن بالثانية
-    long ageInSeconds = 420480000;
+    long ageInSeconds = 420480000; // 0.75/0.75
 
-    return 1;
+    return 0;
 }
-
-
-
-
-
-//
-//int main()
-//{
-//
-//
-//
-//
-//
-//
-//
-//
-//   //cout << " *********************************\n";
-//   //cout << " *********************************\n";
-//   //cout << " *********************************\n";
-//   //cout << " *********************************\n";
-//
-//
-//    //cout << "*     *\n";
-//    //cout << "*     *\n";
-//    //cout << "* * * *\n";
-//    //cout << "*     *\n";
-//    //cout << "*     *\n";
-//    ////////////////////////////////////
-//  /* cout << " *********************************\n";
-//   cout << " name: Hajar saeed AbduAziz\n";
-//   cout <<" age:39 years\n";
-//   cout << " city : Ciro\n";
-//   cout << " country: Egypt\n";
-//   cout << " *********************************\n";
-//
-//   cout << "I Love Programming \n\nI promise to be the best develober ever !\n\n";
-//   
-//   cout << "I know it will take sometime to practice, but i will achieve my goal \n\n";
-//   cout << "Best regards\n Hajar saeed makhlof";*/
-//  
-//
-//    /*
-//   
-//    cout << "please enter rectangle length ,  width: \n";
-//    int width , length ;
-//   
-//  
-//    cin >> length >> width ;
-//
-//   
-//
-//    cout << "the area of rectangle with length  "<<length << " and width "<< width << " = " << length * width;
-//
-//    */
-//
-// 
-//
-//    //firstQuestion();
-//   // secondQuestion();
-//  //  question7();
-//   // question9();
-//   // question10();
-//   // question14();
-//   // question15();
-//   // question17();
-//   // question19();
-//   // question20();
-//    //question21();
-//   // question22();
-//   // question31();
-//    //question35();
-//   // question39();
-//   // question1();
-//   // question40();
-//    //question42();
-//  //question43();
-//    // question47();
-//    // question48();
-//
-//
-//
-//   
-//   
-// 
-//
-//   
-//
-//
-//}
-
+// great work 😎 9.5/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [ ] formatted code to be clear and readable
- [x] commented some phrases for blocks of code